_posts/2022-02-17-quarkus-superheroes-to-the-rescue.adoc

```diff
@@ -0,0 +1,239 @@
+
+The Event Statistics service contains a full suite of unit and integration tests using https://quarkus.io/guides/getting-started-testing#further-simplification-with-injectmock[mocking^] and https://quarkus.io/guides/getting-started-testing#using-spies-instead-of-mocks-with-injectspy[spying^]. It also shows how to https://quarkus.io/guides/getting-started-testing#testing-dev-services[inject Dev Services information^] to configure test resources, such as a `+KafkaProducer+`.
+
+There are four https://quay.io/repository/quarkus-super-heroes/event-statistics?tab=tags[container images available for the Event Statistics service^]: `+java11-latest+`, `+java17-latest+`, `+native-java11-latest+`, and `+native-java17-latest+`.
+
+=== Super Hero UI
+
+The Super Hero UI application, shown in Figure 1 above, is an http://angular.io/[Angular^] single-page application deployed via Node.js. It communicates to the Fight service via REST. The focus of the sample application is on the Quarkus applications and not the Angular UI.
+
+A single https://quay.io/repository/quarkus-super-heroes/ui-super-heroes?tab=tags[container image is available for the Super Hero UI^]: `+latest+`.
+
+== Run the Application Locally
+
+There are a few ways to run the application locally depending on what your goals are. https://docs.docker.com/compose[Docker Compose^] can be used if you want to run the <<running-the-entire-application,entire application>>, or a <<running-a-subset-of-the-application,subset of it>>, using the https://quay.io/quarkus-super-heroes[pre-built container images^]. Alternatively, each application can be <<running-locally-via-quarkus-dev-mode,run from source>>.
+
+=== Running Locally via Docker Compose
+
+The entire application or a subset of it can be started with a single Docker Compose command.
+
+==== Running the Entire Application
+
+The https://github.com/quarkusio/quarkus-super-heroes/tree/main/deploy/docker-compose[`+deploy/docker-compose+`^] directory in the https://github.com/quarkusio/quarkus-super-heroes[root of the respository^] contains compose files for each of the four versions of the application: JVM 11, JVM 17, native built with Java 11, and native built with Java 17. Additionally, https://prometheus.io[Prometheus^] monitoring can be started with the https://github.com/quarkusio/quarkus-super-heroes/blob/main/deploy/docker-compose/prometheus.yml[supplied `+prometheus.yml+` file^].
+
+From the +quarkus-super-heroes+ directory, simply run the command based on the version of the application you want to run, as specified in Table 1:
+
+.Docker Compose commands
+|===
+|Application Version|Docker Compose Command
+
+|JVM Java 11|`+docker-compose -f deploy/docker-compose/java11.yml -f deploy/docker-compose/prometheus.yml up+`
```

Comment:
too much detail imo for a intro blog I would just pick one variant and point to main repo docs for details

Reply:
I removed some stuff (all the tables).